plugins/tags/plugin.cpp

```diff
@@ -203,10 +205,18 @@ namespace golos { namespace plugins { namespace tags {
         boost::program_options::options_description&,
         boost::program_options::options_description& config_file_options
     ) {
+        config_file_options.add_options()
+            (
+                "tags-number", boost::program_options::value<uint16_t>()->default_value(5),
+                "Maximum number of tags"
+            ) (
+                "tag-max-length", boost::program_options::value<uint32_t>()->default_value(512),
```

Comment:
uint16_t is enough, it can not fit in block if larger : )
btw, is there reason to have such large tags? whey become unusable for categorisation, maybe good in some inproper usages, idk. for example, stackoverflow limits tags to 35 symbols and it's ok.